ftprime/argrecorder.py

```diff
@@ -315,7 +329,7 @@ def tree_sequence(self, samples=None):
                                  sites=self.sites, mutations=self.mutations)
         #                        migrations=self.migrations)
         sample_nodes = self.get_nodes(samples)
-        return ts.simplify(samples=sample_nodes)
+        return ts.simplify(samples=sample_nodes, sequence_length=self.sequence_length)
```

Comment:
This won't work because (at least in current master 503704e332e0dede3d6351ce74ba49cff27ffe6b ) msprime.TreeSequence.simplify() does not include this argument